### ISSUE TITLE
Enter personal information

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#7]
## User story
1. As a user (HR employee)
2. I want / need to enter the name, surname, birthday, e-mail address of the employee
3. So that the personal information of the user can be saved

## Acceptance criteria
* 
* 
* 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
@TamaraHoock @veitmo 
After our meeting with Adorsys: The SDs think we should add the following data columns:
- *team* (e.g. L4)
- *user_role* (e.g. guest, employee, admin)
-  *company* (e.g. FAU, Adorsys)

Do you agree? 
Which of these should be optional?

**Comment by Idontker** [comment]
- Backend: @jackDS008, @Jean28518 
- FrontEnd: @annikakrause, @valentinBraeutigam, @steve-237 @Idontker

**Comment by veitmo** [comment]
> 

@Idontker 
Thanks for your suggestions.

- team: I agree (only needed if user_role = "employee", otherwise optional)
- user_role: I agree --> default selection should be "employee"
- company: could make sense in a later stage especially in case of user_role="employee" --> optional

**Comment by Idontker** [comment]
@TamaraHoock @veitmo 
I am not sure of the meaning of the following accepting criteria. 

> - Automatic initialization of "status" -field with "Account initialized"

Can you explain the use/meaning of the status field in a registering process?

**Comment by Idontker** [comment]
@TamaraHoock @veitmo
I have got another question. Should an admin be able to register a user as an admin via the web application?

**Comment by veitmo** [comment]
In general, a status field should be assigned for each digital identity, so that one can later see if the digital identity is active/inactive...
In the case of the registering-process the status assigned to the digital identity under creation should be "Initialized". It should only later in the process be changed to active, as soon as the user has provided all necessary information and the final token has been issued. I hope that answers your question.

**Comment by veitmo** [comment]
@TamaraHoock @Idontker 
yes, this should be possible

**Comment by Idontker** [comment]
@veitmo could you update the user story/acceptance criteria with the new information? Otherwise the sprint review of this story might be harder
